### PR TITLE
Add linkwrapper and update cards

### PR DESCRIPTION
**Pull request by drewdrewthis**

### CODE REVIEW

File: styleguide/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
```
Comment (cisacke) [inline styleguide/index.html]: Do we want this?

File: src/organisms/cards/AddCard/index.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ function renderContent( props ) {
   var txt, icon;
 
   if ( beneficiary ) {
-    txt = 'Add Beneficiary';
+    txt = 'Add person to policy';
```
Comment (cisacke) [inline src/organisms/cards/AddCard/index.js]: outside the scope of this PR but I'm still not sure about this component. I was looking at the designs & if you look at 2.4 in the flow, the text changes when you have already added a category, which means we'd need 2 types of component for each state. Like you mentioned last night, the trade off is having to specify `txt` and `icon` when you go to implement the component, but I think this is preferable to having to add all the conditionals for each possible state.

File: src/atoms/LinkWrapper/LinkWrapper.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <a
+      { ..._.omit(props, ['children', 'className']) }
+      className={ classnames(className, styles.link) }>
+      { children }
+    </a>
+  )
+}
+
+LinkWrapper.propTypes = {
```
Comment (cisacke) [inline src/atoms/LinkWrapper/LinkWrapper.js]: Different prop types needed here?

File: src/atoms/LinkWrapper/LinkWrapper.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+function LinkWrapper ( props ) {
+  const {
+    className,
+    children,
+  } = props;
+
+  return (
+    <a
+      { ..._.omit(props, ['children', 'className']) }
```
Comment (cisacke) [inline src/atoms/LinkWrapper/LinkWrapper.js]: is the only prop you're using here `href`? if so you could declare that explicitly to help with readability

File: package.json [hunk 5]
```diff
@@ -4,6 +4,7 @@
   "description": "React / Webpack / Babel framework for quick set up",
   "main": "index.js",
   "scripts": {
+    "open": "open http://localhost:6060",
```
Comment (cisacke) [inline package.json]: 🙌

File: src/atoms/LinkWrapper/LinkWrapper.js (hunk 4 above)
Comment (drewdrewthis) [inline src/atoms/LinkWrapper/LinkWrapper.js]: @cisacke It's passing through any standard anchor tag props, so the user could set `target`, `rel`, `href`, whatever.

File: src/atoms/LinkWrapper/LinkWrapper.js (hunk 3 above)
Comment (drewdrewthis) [inline src/atoms/LinkWrapper/LinkWrapper.js]: wooops, good catch. Thanks @cisacke

File: src/organisms/cards/AddCard/index.js (hunk 2 above)
Comment (drewdrewthis) [inline src/organisms/cards/AddCard/index.js]: Make sense.

**Pull request merged**